go.mod

```diff
@@ -389,4 +389,8 @@ replace (
 
 	// until merged upstream: https://github.com/mwitkow/grpc-proxy/pull/69
 	github.com/mwitkow/grpc-proxy => github.com/smartcontractkit/grpc-proxy v0.0.0-20230731113816-f1be6620749f
+	github.com/smartcontractkit/chainlink-cosmos => github.com/smartcontractkit/chainlink-cosmos v0.0.0-20231102020720-36bfe1e78327
+	github.com/smartcontractkit/chainlink-relay => github.com/smartcontractkit/chainlink-relay v0.0.0-20231107215547-ee2e975fb9d9
+	github.com/smartcontractkit/chainlink-solana => github.com/smartcontractkit/chainlink-solana v0.0.0-20231101173529-89dcfddba1e5
+	github.com/smartcontractkit/chainlink-starknet/relayer => github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20231102015854-0056a86b5f79
```

Comment:
If you use these directly, instead of replacing them here, then they will be automatically used in the other modules, as well as the integrations tests.
```suggestion
```

Reply:
I don't really understand what you mean by this. This replace reffered to chainReader PRs in other repos right? Why would it automatically refer to those now?

Reply:
Oh, nice! Thanks for the tip